robot_driver/src/estimators/comp_filter_estimator.cpp

```diff
@@ -0,0 +1,142 @@
+#include "robot_driver/estimators/comp_filter_estimator.h"
+
+CompFilterEstimator::CompFilterEstimator() {}
+
+void CompFilterEstimator::init(ros::NodeHandle nh) {
```

Comment:
I was trying to avoid making this class ros dependent but I guess it would get ugly so ok for now